plugins/richcombo/plugin.js

```diff
@@ -96,6 +96,13 @@ CKEDITOR.plugins.add( 'richcombo', {
 				panelDefinition: panelDefinition,
 				items: {}
 			};
+
+			/**
+			 * Array containing richCombos registered listeners.
+			 *
+			 * @private
+			 */
```

Comment:
There is missing type of this property. It'd be ideal to use dedicated type (see #2383), but some more generic approach would be also enough.